README.md

```diff
@@ -46,57 +40,40 @@ We've set you up with a build based on [our custom fork](https://backpack.github
 
-## API implementation
-
-We're hitting a real Skyscanner API in this test, and have provided a basic - but incomplete - Node.js client on top of it to make things easier for you.
-
-The [`server.js`](./server/src/server.js) file starts an [Express](https://expressjs.com/) server with a couple of routes. To run the server:
-
-**NB:** If you'd prefer not to use Node.js, that's fine. Feel free to copy from it, or roll your own.
+## Backend implementation
 
-The underlying Skyscanner [API documentation is available here](https://skyscanner.github.io/slate/#flights-live-prices) and a [test harness is provided](http://business.skyscanner.net/portal/en-GB/Documentation/FlightsLivePricingQuickStart) for you to try queries out.
+We've provided a basic - but incomplete - Node.js client located in the `server` folder.
 
-You can use the Skyscanner `sky` location schema, and the `EDI-sky` and `LOND-sky` placenames in your query.
+The [`server.js`](./server/src/server.js) file starts an [Express](https://expressjs.com/) server with a premade endpoint. To run the server:
 
-**NB:** You should ensure the [http://localhost:4000/api/search](http://localhost:4000/api/search) route accepts parameters from your front end, and returns results to the front end in an appropriate format.
-
-The API will return collections of different items:
+The server endpoint will return collections of different items:
 
 * **Itineraries** - These are the containers for your trips, tying together **Legs**, and **prices**. Prices are offered by an **agent** - an airline or travel agent.
 
 * **Legs** - These are journeys (outbound, return) with **duration**, and **carriers**<sup>[1](#footnote1)</sup>. These contain one or more **Segments** and **stops**.
 
-* **Segments** - Individual flight information with directionality.
-
 A good structure to represent trip options would be hierarchical:
 
 ```
 Itineraries
   Legs
-    Segments
 ```
 
-Your key will be rate-limited to ~5 queries per minute.
-
 ## Running the project
 
-The project is configured to run in [Docker](https://docs.docker.com/get-started/) using [Docker Compose](https://docs.docker.com/compose/gettingstarted/) to connect the client and server containers.
+To startup the frontend client and server run the following command to run the solution in the background.
 
-- `APIKEY=<key> docker-compose up --build`
-
-Feel free to develop the project as you want (in or out of Docker), but please ensure that the project works as expected inside Docker when you're ready to submit.
+`npm start`
 
 ## Submission Guidelines
 
-* Please submit your program by sending a zip file to your Skyscanner contact.
-
+### `TODO: Update with the hired platform submission steps`
 * The zip file should be named {yourname}.zip, and should itself contain the full-stack-recruitment-test project folder with your submission.
 
 * The zip file should contain the [FOLLOW-UP.md](./FOLLOW-UP.md) file with answers to the follow-up questions.
```

Comment:
Should we give guidelines on how much time/content we expect in follow_up.md? If we're moving candidates towards a soft time limit and we are still grading on follow_up.md to me it feels sensible at least for us to have an opinion on how long we expect candidates to spend filling this in. Something like

_We would like to know the reasoning behind the technical choices you made in completing the project, please spend around 10 minutes completing follow_up.md_

Reply:
Yes we should add something like this to give a rough estimate, I need to also check how this works with the hired platform in this scenario to see if it goes here or direct on the platform 😄